Day1/day1_p2.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const data = require('./day1_p2');
+const fs = require('fs');
+const arr = fs
+  .readFileSync('/Users/Elena/AdventOfCode/Day1/input_01.txt', 'utf-8')
+  .split('\n')
+  .map(x => parseInt(x));
```

Comment:
When testing the code, you don't need to work with real data.
Provide a short set of data and use it, like in `Day1/day1_p1.test.js`